metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1131,7 +1132,19 @@ def from_api(  # noqa: C901
                 logger.warning(
                     f"Failed to extract explore {explore_name} from model {model}: {e}"
                 )
-
+        except DeserializeError as e:
+            logger.warning(
```

Comment:
the reporter.warning line will automatically print a log line, no need to do it explicitly